build.cake

```diff
@@ -131,25 +150,44 @@ Task("Test")
 	.IsDependentOn("Build")
 	.Does(() =>
 	{
-		NUnit3(UNIT_TEST_PATH);
+		NUnit3(BIN_DIR + UNIT_TEST_ASSEMBLY);
 	});
 
 //////////////////////////////////////////////////////////////////////
 // PACKAGE
 //////////////////////////////////////////////////////////////////////
 
-Task("Package")
+Task("PackageNuGet")
 	.IsDependentOn("Build")
 	.Does(() => 
 	{
-		CreateDirectory(PACKAGE_DIR);
-
-        NuGetPack(NUSPEC_PATH, new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = BIN_DIR,
-            OutputDirectory = PACKAGE_DIR
-        });
+        NuGetPack("nuget/nunit-project-loader.nuspec",
+			new NuGetPackSettings()
+			{
+				Version = nugetVersion ?? packageVersion,
+				OutputDirectory = OUTPUT_DIR,
+				Files = new [] {
+					new NuSpecContent { Source = "../LICENSE.txt" },
+					new NuSpecContent { Source = binaries + "nunit-project-loader.dll", Target = "tools" }
+				}
+			});
+	});
+
+Task("PackageChocolatey")
+	.IsDependentOn("Build")
+	.Does(() =>
+	{
+		ChocolateyPack("choco/nunit-project-loader.choco.nuspec",
```

Comment:
Have you noticed the nupkg isn't taking the `.choco.` suffix? Looks like it's uses the package id + version instead?

Maybe best to file rename after packing to get this, if you wanted it? @ferventcoder suggested that wasn't good for choco on https://github.com/nunit/nunit-console/issues/253#issuecomment-315315820 however, maybe best to see how that gets resolved first.

If the solution is not to do it, it would be good to remove the `.choco` from the nuspec filename.